src/dotnet-grpc/runtimeconfig.template.json

```diff
@@ -0,0 +1,3 @@
+{
+  "rollForwardOnNoCandidateFx": 2
```

Comment:
Can you add a comment of what this does

Reply:
Not in this file? I'll add it in the csproj.